src/main/java/org/primefaces/component/datatable/DataTable.java

```diff
@@ -305,32 +310,32 @@ public void processUpdates(FacesContext context) {
             setSelection(null);
         }
 
-        Map<String, FilterMeta> filterBy = getFilterBy();
-        if (!filterBy.isEmpty()) {
-            ELContext elContext = context.getELContext();
-            for (FilterMeta filter : filterBy.values()) {
-                UIColumn column = filter.getColumn();
-                if (column == null) {
-                    column = findColumn(filter.getColumnKey());
-                    filter.setColumn(column);
-                }
-
-                if (column != null) {
-                    ValueExpression columnFilterValueVE = column.getValueExpression(Column.PropertyKeys.filterValue.toString());
-                    if (columnFilterValueVE != null) {
-                        if (column.isDynamic()) {
-                            DynamicColumn dynamicColumn = (DynamicColumn) column;
-                            dynamicColumn.applyStatelessModel();
-                            columnFilterValueVE.setValue(elContext, filter.getFilterValue());
-                            dynamicColumn.cleanStatelessModel();
-                        }
-                        else {
-                            columnFilterValueVE.setValue(elContext, filter.getFilterValue());
-                        }
-                    }
-                }
-            }
-        }
+//        Map<String, FilterMeta> filterBy = getFilterBy();
+//        if (!filterBy.isEmpty()) {
+//            ELContext elContext = context.getELContext();
+//            for (FilterMeta filter : filterBy.values()) {
+//                UIColumn column = filter.getColumn();
+//                if (column == null) {
+//                    column = findColumn(filter.getColumnKey());
+//                    filter.setColumn(column);
+//                }
+//
+//                if (column != null) {
+//                    ValueExpression columnFilterValueVE = column.getValueExpression(Column.PropertyKeys.filterValue.toString());
+//                    if (columnFilterValueVE != null) {
+//                        if (column.isDynamic()) {
+//                            DynamicColumn dynamicColumn = (DynamicColumn) column;
+//                            dynamicColumn.applyStatelessModel();
+//                            columnFilterValueVE.setValue(elContext, filter.getFilterValue());
+//                            dynamicColumn.cleanStatelessModel();
+//                        }
+//                        else {
+//                            columnFilterValueVE.setValue(elContext, filter.getFilterValue());
+//                        }
+//                    }
+//                }
+//            }
+//        }
```

Comment:
Why comment out and not just remove?

Reply:
Because I'm still trying to understand what it's suppose to do